### PR TITLE
Added prerequisite section

### PR DESCRIPTION
**Pull request by AustinSeto**
Addresses #57 by asking the user to register for a Github account if they do not already have one

### CODE REVIEW

**Review by griffinhadfield — APPROVED** [review]
Looks good in LGDEV.

File: README.adoc [hunk 1]
```diff
@@ -58,6 +55,10 @@ hello.html
 include::finish/src/main/webapp/hello.html[]
 ----
 
+== Additional prerequisites
+
+Before you begin, a Github account is required for the examples in this guide. 
+https://github.com/join[Register for a Github account^] if you do not already have one.
```
Comment (Charlotte-Holt) [inline README.adoc:61]: Suggested update

>Before you begin, a Github account is required for the examples in this guide. 
Register for a Github account if you do not already have one. 

To

>Before you begin, you must have a GitHub account to complete this guide. 
Register for a GitHub account, if you don't already have one.

**Review by Charlotte-Holt — COMMENTED** [review]
@AustinSeto Just a few suggested changes. Thx!

File: README.adoc (hunk 1 above)
Comment (AustinSeto) [reply README.adoc]: Update implemented

**Review by Charlotte-Holt — APPROVED** [review]

**Pull request merged**